caffe2/contrib/aten/aten_op_template.h

```diff
@@ -89,16 +89,18 @@ class ATenOp : public Operator<Context> {
     CAFFE_THROW("Unknown type meta"); // TODO: improve error message...
   }
   void assignTo(Tensor* dst, const at::Tensor& src_) {
-    at::Tensor src = src_.contiguous();
-    auto at_sizes = src.sizes();
+    at::Tensor* src = new at::Tensor(src_.contiguous());
```

Comment:
I am not 100% sure what exactly you want to do here, but if it's a `Tensor` you want as context, you should directly use `TensorImpl*`. `unsafeReleaseTensorImpl`, and then `c10::raw::intrusive_ptr::decref()` inside the deleter.

Reply:
I just want to make sure I'm changing the code in such a way that at::DataPtr is constructed appropriately.